source/_integrations/apple_tv.markdown

```diff
@@ -10,33 +10,126 @@ ha_release: 0.49
 ha_domain: apple_tv
 ha_codeowners:
   - '@postlund'
+  - '@wizbangcrash'
 ha_config_flow: true
+ha_zeroconf: true
+ha_platforms:
+  - remote
+  - media_player
 ---
 
-The Apple TV integration allows you to control an Apple TV (any generation). See the
-[remote platform](/integrations/apple_tv#remote) if you want to send remote control buttons,
-e.g., arrow keys.
+The Apple TV integration allows you to control an Apple TV (any generation).
 
-There is currently support for the following device types within Home Assistant:
+There is currently support for the following entities within the Apple TV device:
 
-- Media Player
+- [Media Player](#media_player)
 - [Remote](#remote)
 
-## Configuration
+## Media Player
+
+The Apple TV media player platform will create a Media Player entity for each
+Apple TV discovered on your network.
+This entity will display the active app and playback controls.
+
+## Remote
+
+The Apple TV remote platform will automatically create a Remote entity for each Apple TV
+configured on to your Home Assistant instance.
+These entities allow you to turn the device on/off and to send control commands.
 
-Menu: *Configuration* > *Integrations*
+The following commands are currently available:
+
+- `wakeup`
+- `home`
+- `home_hold`
+- `top_menu`
+- `menu`
+- `select`
+- `up`
+- `down`
+- `left`
+- `right`
+- `volume_up`
+- `volume_down`
 
-Press on **Apple TV** and configure the integration:
+**NOTE:** Not all commands are supported by all Apple TV versions
 
-* Enter either an IP address or a device name and follow the next few steps
+### Service `send_command`
+
+| Service data<br>attribute | Optional | Description  |
+| ------------------------- | -------- | ------------ |
+| `entity_id`               | no       | `entity_id` of the Apple TV |
+| `command`                 | no       | Command, or list of commands to be sent |
+| `num_repeats`             | yes      | Number of times to repeat the commands |
+| `delay_secs`              | yes      | Interval in seconds between one send and another <br> This is a `float` value e.g. 1, 1.2 etc. |
+
+**Examples**
```

Comment:
This is upstream documentation from the remote platform, and should not be duplicate documented.

Instead, you could link to the remote service and link it up. On this location, just showing examples should be sufficient.

```suggestion
### Service `remote.send_command` examples
```

Reply:
Can you point me to the remote service link you suggest as I have spent a fair bit of time scanning the documentation site and have been unable to find any information on `num_repeats` or `delay_secs` on anything other than the `broad link`, `harmony` & `xbox` integration pages. If I type "delay_secs" into the Documentation Search box it responds with nothing.
I'm fairly new to Home Assistant and I find one of the biggest struggles is finding enough documentation to stop me having to revert to scanning the source code. I also find the examples, in most cases, assume the user is an advanced user and only show the few lines relevant to the specific feature being added and rarely give a bigger picture.
I'd prefer to put a link to the actual documentation about the remote service and its supported keys than leave the user to do what I had to do 😄 
Not complaining. Just want to help others in my situation.

Reply:
In my search I realised that the "Service send_command" section you are asking me to remove is repeated in the Broadlink, Xbox and Harmony integration's documentation.
I would agree that it would be better to have a generic Remote page that covers the attributes a remote should have and then each integration lists the features it currently supports.
Happy to help do this if it make sense to you.

Reply:
I see, it should have been documented here: <https://www.home-assistant.io/integrations/remote>


For now, let us keep it in here until we fix our platform pages. 👍